tests/config/database.php

```diff
@@ -15,6 +17,18 @@
             'database' => env('MONGO_DATABASE', 'unittest'),
         ],
 
+        'mongodb_repl' => [
+            'name'      => 'mongodb_repl',
+            'driver'    => 'mongodb',
+            'host'      => $mongoReplHost,
+            'port'      => $mongoReplPort,
+            'database'  => env('MONGO_DATABASE', 'unittest'),
+            'options'   => [
+                'replicaSet'                => 'rs',
+                'serverSelectionTryOnce'    => false,
+            ],
+        ],
+
         'dsn_mongodb' => [
```

Comment:
The `options` element containing the `replicaSet` option can be added here (and all other connections to MongoDB)

Reply:
Since this is a PHP file, do you think it makes sense to define the options array once at the top of the file and reference it in the three times it's used herein?